tests/test_SklearnRandomForestConverters.py

```diff
@@ -30,11 +31,37 @@ def test_random_forest_classifier(self):
                                      allow_failure="StrictVersion(onnx.__version__) < StrictVersion('1.2') or "
                                                    "StrictVersion(onnxruntime.__version__) <= StrictVersion('0.2.1')")
 
+    @unittest.skipIf(not onnx_built_with_ml(),
+                     reason="Requires ONNX-ML extension.")
+    def test_random_forest_classifier_mismatched_estimator_counts(self):
```

Comment:
Could you add some elaborate tests where you can verify if the converter works fine? Look at this: https://github.com/onnx/sklearn-onnx/blob/master/tests/test_SklearnMLPConverter.py

Reply:
Hi @prabhat00155 , is that a general request for what I could do next to help? Or is not having more elaborate tests blocking your approval for this particular PR for some reason?

Reply:
I just feel that would be a more comprehensive test. I don't think fitting on 3 data points and predicting on the same checks the correctness of the converter properly. Anyways, I think you are mainly focussed on testing your fix, so I'll approve your PR.